### DISCUSSION TITLE
BTT-SD-TF-Cloud-V1.0

### DISCUSSION THREAD

**Discussion by kacharros** [luc-github/ESP3D#635] [F.A.Q] [answered]
Podría instalar en BTT-SD-TF-Cloud-V1.0 y usar ESP3D WEBUI?

**Comment by luc-github** [comment, marked as answer]
No, ahora no es possible
para mi, no es una buena solucion  : https://github.com/luc-github/ESP3D/issues/454

**Reply by kacharros** [reply]
Gracias.
